getting-started/organizations.mdx

```diff
@@ -41,6 +41,99 @@ you can configure this by changing the organization metaname.
 This setting will require all users within an organization to have 2FA enabled before they can use your product.
 We proactively email users who are not yet enrolled in 2FA, and allow your customers to set a deadline for when they must enroll.
 
+### Maximum Users Per Org
+
+A simple way to achieve more organization customization this is through our Backend API endpoints. On all of our libraries, we have the **Update Organization** endpoint, 
+which takes in as one of its parameters a maximum number of users in the organization. This means through your product, you can programatically 
+adjst how many users can be in each organization based on your own requirements. 
+
+As an example, let's say you have an integration with some kind of paid API service, that you only want used by your higher tiered customer plans. 
+By using the **Update Organization** endpoint, you could conditionally set a cap on users for specific tiers of customers, so that you 
+don't have to manually cheeck and limit through the dashboard.
+
+<Tabs>
+  <Tab title="Express">
+    ```js
+    const orgUpdateObject = {
+        orgId: '0eca0135-5daf-46ae-a704-04fed3be62a0',
+        name: 'Test Org',
+        maxUsers: 5
+    }
+
+    auth.updateOrg(orgUpdateObject)
+    ```
+  </Tab>
+  <Tab title="Node">
+    ```js
+    const orgUpdateObject = {
+        orgId: '0eca0135-5daf-46ae-a704-04fed3be62a0',
+        name: 'Test Org',
+        maxUsers: 5
+    }
+
+    auth.updateOrg(orgUpdateObject)
+    ```
+  </Tab>
+  <Tab title="Flask">
+    ```python
+      auth.update_org_metadata('0eca0135-5daf-46ae-a704-04fed3be62a0', 'Test Org', can_setup_saml=None, metadata=None, max_users=5)
```

Comment:
These can just be:
`auth.update_org_metadata('0eca0135-5daf-46ae-a704-04fed3be62a0', max_users=5)`